skills/gh-address-comments/scripts/ai_triage.py

```
@@ -0,0 +1,175 @@
+        parser.error("Provide --input FILE or --pr NUMBER")
+
+    # Load data
+    try:
+        if args.input:
+            with open(args.input) as f:
+                data = json.load(f)
+        else:
+            repo_flag = []
+            if args.repo:
+                repo_flag = ["--repo", args.repo]
+            out = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)